registrations/metrics.py

```diff
@@ -180,6 +181,14 @@ def registrations_source_sum(self, user, start, end):
             .filter(source__user__username=user)\
             .count()
 
+    def change_language_sum(self, user, start, end):
+        return Change.objects\
+            .filter(created_at__gt=start)\
+            .filter(created_at__lte=end)\
+            .filter(source__user__username=user)\
+            .filter(action='change_language')\
+            .count()
```

Comment:
Do we also maybe want to add tests for this function? Since I'm not sure that it's 100% correct, because these functions only get sent `start` and `end`, no other arguments.

We should also create a `Change` model in this test: https://github.com/praekelt/hellomama-registration/blob/develop/registrations/test_metrics.py#L484-L493 , to ensure that the function is named correctly for the system to find it.